apps/openssl.c

```diff
@@ -292,6 +292,7 @@ int main(int argc, char *argv[])
 
     /* If there's a command, run with that, otherwise "help". */
     ret = argc > 0
+            && strcmp(argv[0], "-help") != 0 && strcmp(argv[0], "--help") != 0
```

Comment:
Maybe support also `-h` ?

Reply:
Good thought - I've added support now for `-h`, and for `--h` as well (to be on the safe side).

Reply:
I think this is a mistake because it doesn't easily catch the  case where someone forgets the command name.  I do that, and I doubt I am the only one.

This is a change, but not a needed one as the current behavior points out the mistake and tells what to do to fix it.

Reply:
I disagree, Rich, I many times wanted to display the list of commands that openssl help would print but I typed openssl -h instead which gave me rather useless output. I do not see why this change would be wrong.